src/lava/magma/core/model/nc/model.py

```diff
@@ -38,3 +106,148 @@ def allocate(self, net: Net):
         Note: This should work as before.
         """
         pass
+
+
+class NcProcessModel(AbstractNcProcessModel):
+    def __init__(self, proc_params: ty.Dict[str, ty.Any],
+                 board: ty.Type[NxBoard],
+                 loglevel=logging.WARNING):
+        super(AbstractNcProcessModel, self).__init__(proc_params,
+                                                     loglevel=loglevel)
+        self.board = board
```

Comment:
I understand this is probably mock code at the moment. However, just mentioning that NcProccessModel will work with the Net Interface and should not have the board object. Please remove it from here.

Reply:
ACK. That was not entirely clear to me.